tests/t208-elemrestriction-f.f90

```diff
@@ -2,62 +2,94 @@
       program test
       implicit none
       include 'ceed/fortran.h'
-
+  
       integer ceed,err
       integer x,y
-      integer i,r
-
-      integer ne
-      parameter(ne=8)
-      integer blksize
-      parameter(blksize=5)
-
-      integer*4 ind(2*ne)
+      integer r
+      integer i,j,k
+  
+      integer ne,elemsize,nb,blksize
+      parameter(ne=8,elemsize=2,nb=2,blksize=5)
+      integer ind(elemsize*ne)
+      integer layout(3)
+      integer blk,elem,indx
+  
       real*8 a(ne+1)
-      integer*8 aoffset
-
+      real*8 yy(nb*blksize*elemsize)
+      real*8 xx(ne+1)
+      real*8 diff
+      integer*8 aoffset,xoffset,yoffset
+  
       character arg*32
-
+  
       call getarg(1,arg)
       call ceedinit(trim(arg)//char(0),ceed,err)
-
+  
       call ceedvectorcreate(ceed,ne+1,x,err)
-
+  
```

Comment:
Are these adding trailing whitespace?

Reply:
Oh, that was my IDE adding bad extra spaces. I'll fix that

Reply:
Fixed